src/sources/aws_s3/sqs.rs

```diff
@@ -513,18 +522,53 @@ impl IngestorProcess {
         let mut stream = lines.filter_map(move |line| {
             let mut log = LogEvent::from_bytes_legacy(&line).with_batch_notifier_option(&batch);
```

Comment:
As you mentioned, this should probably use `BytesDeserializer::parse` to format this in the correct format depending on the log namespace